go/enclave/limiters/rolluplimiter.go

```diff
@@ -27,13 +24,14 @@ func NewRollupLimiter(size uint64) RollupLimiter {
 }
 
 // todo (@stefan) figure out how to optimize the serialization out of the limiter
-func (rl *rollupLimiter) AcceptBatch(encodable interface{}) (bool, error) {
-	encodedData, err := rlp.EncodeToBytes(encodable)
+func (rl *rollupLimiter) AcceptBatch(batch *core.Batch) (bool, error) {
```

Comment:
That's iffy. If the assumption with the compression factor does not hold we end up with unsubmittable batch that we have no way to get rid of right now

Reply:
increased the compression to 85%. imo, it's pretty much impossible not to get that